content/techniques/file-upload.md

```diff
@@ -4,6 +4,14 @@ To handle file uploading, Nest provides a built-in module based on the [multer](
 
 > warning **Warning** Multer cannot process data which is not in the supported multipart format (`multipart/form-data`). Also, note that this package is not compatible with the `FastifyAdapter`.
 
+#### Use with Express (default)
+
+If you want to use the `Express.Multer.File` type for `@UploadedFile()` files, it is necessary to install the types:
+
+```shell
+$ npm i -D @types/multer
+```
+
```

Comment:
Instead of adding this as a heading, I think we should rather add a blockquote (hint) + the heading title is somewhat misleading, especially the "default" part given that this doesn't work with Fastify at all (and after reading this heading, I would have assumed there's an alternative way covered)

Reply:
For me it is indifferent, @jmcdo29  what do you think?